var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -148,6 +152,7 @@ def url_for_version(self, version):
         return url
 
     def setup_environment(self, spack_env, run_env):
+        spack_env.set('MAKEFLAGS', '-j{0}'.format(make_jobs))
```

Comment:
How much of a speed improvement does this give you? Qt has always been one of the most painful packages to compile so hopefully this helps.